src/ophys_etl/pipelines/brain_observatory/scripts/run_mesoscope_splitting.py

```diff
@@ -124,8 +187,38 @@ def split_image(input_tif, experiments, name):
     return outs, meta
 
 
-def split_timeseries(input_tif, experiments, **h5_opts):
-    mt = MesoscopeTiff(input_tif)
+def split_timeseries(input_tif: MesoscopeTiff,
+                     experiments: List[Dict],
+                     **h5_opts) -> Tuple[Dict, Dict]:
+    """Takes a timeseries file from a Mesoscope ophys session and a list
+    of the experiments performed during that session and splits the data
+    into a timeseries file (as a .h5) for each individual experiment.
+
+    Parameters
+    ----------
+    input_tif : MesoscopeTiff
+        MesoscopeTiff object containing the timeseries data that
+        will be split by experiment
+
+    experiments : List[Dict]
+        A list of dictionaries, each containing information about
+        the experiments performed during the ophys session
+
+    **h5_opts
+        Keyword arguments to be used when writing the data to files.
+        If any keyword arguments are supplied, it will trigger the
+        use of chunking when saving the files.
+
+    Returns
+    -------
+    outs : dict
+        A dictionary containing specific information about the experiments,
```

Comment:
key names.
if this is repetitive, define a TypedDict so you can reuse it in all these docstrings